docs/Events/bpmn-events/attach.md

```diff
@@ -4,9 +4,28 @@ title: attach
 nav_order: 1
 description: attach event
 permalink: /events/bpmn-events/attach
-last_modified_date: 2020-04-27T17:54:08+0000
+last_modified_date: 2020-09-22T12:00:00+0000
 parent: "BPMN Events"
 grand_parent: "Events"
 ---
 
 # attach
+
+This event is fired by [attachTo API](/general/api-reference/bpmn-js/attachTo) after bpmn-js container was attached to the [configured DOM node](/general/getting-started/bpmn-js). 
```

Comment:
Well I'm no expert but `is attached to the` sounds more correct rathar than `was attached to the`, or even `gets attached to the`. No big deal though.